stdlib/typing_extensions.pyi

```diff
@@ -569,8 +562,70 @@ else:
     ReadOnly: _SpecialForm
     TypeIs: _SpecialForm
 
+# TypeAliasType was added in Python 3.12, but had significant changes in 3.14.
+if sys.version_info >= (3, 14):
```

Comment:
I made an exception for our "next Python version" rule, as technically typing_extensions is not part of the stdlib.